packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1551,12 +1558,16 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
   ///    sheet.
   ///  * [Scaffold.of], for information about how to obtain the [ScaffoldState].
   ///  * <https://material.io/design/components/sheets-bottom.html#standard-bottom-sheet>
-  PersistentBottomSheetController<T> showBottomSheet<T>(WidgetBuilder builder) {
+  PersistentBottomSheetController<T> showBottomSheet<T>(WidgetBuilder builder, {
+    Color color,
+    double elevation,
+    ShapeBorder shape,
+  }) {
```

Comment:
Similarly, here I think this should be formatted as:

```dart
  PersistentBottomSheetController<T> showBottomSheet<T>(
    WidgetBuilder builder, {
    Color color,
    double elevation,
    ShapeBorder shape,
  }) {
```

Reply:
Done